webhook/resourcesemantics/validation/validation_admit_test.go

```diff
@@ -76,6 +79,29 @@ var (
 		}: &InnerDefaultResource{},
 	}
 
+	callbacks = map[schema.GroupVersionKind]Callback{
+		{
+			Group:   "pkg.knative.dev",
+			Version: "v1alpha1",
+			Kind:    "Resource",
+		}: {
+			SupportedVerbs: map[webhook.Operation]bool{
+				webhook.Create: true,
+				webhook.Update: true,
+			},
+			Callback: resourceCallback},
+		{
+			Group:   "pkg.knative.dev",
+			Version: "v1beta1",
+			Kind:    "Resource",
+		}: {
+			SupportedVerbs: map[webhook.Operation]bool{
+				webhook.Create: true,
+				webhook.Update: true,
+			},
+			Callback: resourceCallback},
+	}
+
```

Comment:
We're doing different layout usually
```suggestion
	callbacks = map[schema.GroupVersionKind]Callback{
		{
			Group:   "pkg.knative.dev",
			Version: "v1alpha1",
			Kind:    "Resource",
		}: {
			SupportedVerbs: map[webhook.Operation]bool{
				webhook.Create: true,
				webhook.Update: true,
			},
			Callback: resourceCallback
		},{
			Group:   "pkg.knative.dev",
			Version: "v1beta1",
			Kind:    "Resource",
		}: {
			SupportedVerbs: map[webhook.Operation]bool{
				webhook.Create: true,
				webhook.Update: true,
			},
			Callback: resourceCallback
		}}
```